components/consumers/dependency-track/main.go

```diff
@@ -47,11 +53,42 @@ func main() {
 	if projectUUID == "" {
 		log.Fatal("project uuid is mandatory for dependency track")
 	}
+	if authURL == "" {
+		log.Fatal("auth url is mandatory for dependency track")
+	}
+	if apiKey == "" {
+		log.Fatal("api key is mandatory for dependency track")
+	}
+	if projectName == "" {
+		log.Fatal("project name is mandatory for dependency track")
+	}
+	if projectVersion == "" {
+		log.Fatal("project version is mandatory for dependency track")
+	}
+
 	c, err := dtrack.NewClient(authURL, dtrack.WithAPIKey(apiKey))
 	if err != nil {
 		log.Panicf("could not instantiate client err: %#v\n", err)
 	}
+
```

Comment:
I'm not a huge fan of this addition, I think it can simplified like this:
```golang
		dtClient, err := dtrack.NewClient(
			authURL,
			dtrack.WithHttpClient(
				 &http.Client{Transport: &http.Transport{
					TLSClientConfig: &tls.Config{
						InsecureSkipVerify: insecure,
					},
				},
			}),
			dtrack.WithDebug(true),
			dtrack.WithAPIKey(apiKey),
		)
		if err != nil {
			log.Panicf("could not instantiate client err: %#v\n", err)
		}
```